### PR TITLE
Changes to RRFS 3- and 13-km domains, setup.sh script bug fixes, make_ics task modification, and tweaks to stochastic physics namelist settings

### PR DESCRIPTION
**Pull request by JeffBeck-NOAA**
## DESCRIPTION OF CHANGES: 
This PR 1) modifies the pre-defined RRFS_NA_3km and RRFS_NA_13km domains to account for changes implemented at EMC and GSL, 2) fixes the bug where the workflow was produced even when DO_ENSEMBLE="FALSE" with ensemble verification turned on, 3) fixes the bug related to shutting of verification tasks when running in NCO mode, 4) sets sotyp_from_climo to "true" in the exregional_make_ics.sh script when using RAP external model data now that operational RAP grib2 files are being used, and 5) tweaks stochastic physics namelist settings to only use what is necessary for each scheme.

## TESTS CONDUCTED: 
The RRFS_NA_3km domain WE2E test was run, then modified and run again using the RRFS_NA_13km domain, both of which passed on Hera (required a change from 6 to 8 nodes for the run_post tasks when running at 3-km resolution due to the increase in domain size).  The setup.sh script bug fixes were tested manually using modifications to the config.sh script, then running the generate step to test behavoir of the added "if" statements.  Changes to the stochastic physics namelist settings were tested manually, since no stochastic physics WE2E test exists yet.  The sotyp_from_climo="true" change was tested as part of my code review for PR [#681](https://github.com/ufs-community/regional_workflow/pull/681). 

## ISSUE (optional): 
Fixes issues [#614](https://github.com/ufs-community/regional_workflow/issues/614) and [#661](https://github.com/ufs-community/regional_workflow/issues/661).

### CODE REVIEW

File: ush/set_predef_grid_params.sh [hunk 1]
```diff
@@ -1167,14 +1167,14 @@ case ${PREDEF_GRID_NAME} in
```
Comment (gsketefian) [inline ush/set_predef_grid_params.sh]: You can remove the commented-out code above here.  It's pretty old now.

**Comment by gsketefian** [comment]
@JeffBeck-NOAA What is the RRFS_NA_13km grid used for?  If for testing, we should probably have a WE2E test for it.

File: ush/set_predef_grid_params.sh (hunk 1 above)
Comment (JeffBeck-NOAA) [reply ush/set_predef_grid_params.sh]: Removed!

**Comment by JeffBeck-NOAA** [comment]
> @JeffBeck-NOAA What is the RRFS_NA_13km grid used for? If for testing, we should probably have a WE2E test for it.

The RRFS_NA_13km grid is just to be able to run a quick test over the full North American RRFS domain.  I can create a WE2E test for it.

**Comment by JeffBeck-NOAA** [comment]
@gsketefian, I just added the RRFS_NA_13km WE2E test.

File: tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_RRFS_NA_13km_ics_FV3GFS_lbcs_FV3GFS_suite_RRFS_v1beta.sh [hunk 2]
```diff
@@ -0,0 +1,54 @@
+#
+# TEST PURPOSE/DESCRIPTION:
+# ------------------------
+#
+# This test is to ensure that the workflow running in community mode
```
Comment (gsketefian) [inline tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_RRFS_NA_13km_ics_FV3GFS_lbcs_FV3GFS_suite_RRFS_v1beta.sh:5]: In the comments, 3km --> 13km and v1alpha --> v1beta.

File: tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_RRFS_NA_13km_ics_FV3GFS_lbcs_FV3GFS_suite_RRFS_v1beta.sh [hunk 3]
```diff
@@ -0,0 +1,54 @@
+#
+# TEST PURPOSE/DESCRIPTION:
+# ------------------------
+#
+# This test is to ensure that the workflow running in community mode 
+# completes successfully on the RRFS_NA_3km grid using the RRFS_v1alpha
+# physics suite with ICs and LBCs derived from the FV3GFS.
+#
```
Comment (gsketefian) [inline tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_RRFS_NA_13km_ics_FV3GFS_lbcs_FV3GFS_suite_RRFS_v1beta.sh:8]: Just wondering how much of this resource tweaking is needed for the 13km vs. 3km grid.

File: tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_RRFS_NA_13km_ics_FV3GFS_lbcs_FV3GFS_suite_RRFS_v1beta.sh (hunk 2 above)
Comment (JeffBeck-NOAA) [reply tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_RRFS_NA_13km_ics_FV3GFS_lbcs_FV3GFS_suite_RRFS_v1beta.sh:5]: Thanks, fixed!

File: tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_RRFS_NA_13km_ics_FV3GFS_lbcs_FV3GFS_suite_RRFS_v1beta.sh (hunk 3 above)
Comment (JeffBeck-NOAA) [reply tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_RRFS_NA_13km_ics_FV3GFS_lbcs_FV3GFS_suite_RRFS_v1beta.sh:8]: I reduced the NNODES for post from 8 to 6, but didn't touch the chgres_cube resources based on what was in the 3-km domain WE2E.  Is that what you're referring to?

Comment (gsketefian) [reply tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_RRFS_NA_13km_ics_FV3GFS_lbcs_FV3GFS_suite_RRFS_v1beta.sh:8]: Yes, I was just wondering which ones you really need vs. just using the defaults.

**Review by willmayfield — APPROVED** [review]

**Pull request merged**